style.scss

```diff
@@ -105,44 +105,59 @@ header {
 main {
     flex-grow: 1;
 
-    .work {
+    .hero-section {
+        padding: 10px;
         background-image: url("./img/man.jpg");
-        background-size: cover;
         background-position: center;
-        height: 400px;
+        background-size: cover;
+        height: 300px;
 
         @media (min-width: $screen-md) {
-            height: 600px;
+            height: 400px;
+        }
+
+        @media (min-width: $screen-lg) {
+            height: 500px;
         }
     }
 
-    .text-about-us {
+    .motto {
         text-align: center;
-        padding-top: 100px;
-        margin: 0 300px 0 10px;
+        text-shadow:
+            3px 3px 6px $dark,
+            0 0 16px $primary,
+            0 0 3px $primary;
+        color: $light;
+        position: absolute;
```

Comment:
It’s not needed here. Just use display: flex for parent component, and then apply vertical alignment for flexbox (align-items).